prerendering.bs

```diff
@@ -425,6 +430,13 @@ Every [=navigable=] has a <dfn for="navigable">loading mode</dfn>, which is one
 
 By default, a [=navigable=]'s [=navigable/loading mode=] is "`default`". A navigable whose [=navigable/loading mode=] is "`prerender`" is known as a <dfn>prerendering navigable</dfn>. A [=prerendering navigable=] that is also a [=top-level traversable=] is known as a <dfn>prerendering traversable</dfn>.
 
+Every [=navigable=] has a <dfn for="navigable">scripting mode</dfn>, which is one of the following:
```

Comment:
My naive question would be: Is this flag associated with `navigable` or something else (e.g. `Document`)?

So, clarifying the context/reasoning for the placement of this flag (no matter of the specific placement) is helpful.

Reply:
Thank you for the discussion! I added a short note trying to cover the points. PTAL :)